Stratis.Bitcoin/Features/BlockStore/LoopSteps/DownloadBlockStep.cs

```diff
@@ -28,28 +28,32 @@ internal DownloadBlockStep(BlockStoreLoop blockStoreLoop)
         }
 
         /// <inheritdoc/>
-        internal override async Task<BlockStoreLoopStepResult> ExecuteAsync(ChainedBlock nextChainedBlock, CancellationToken token, bool disposeMode)
+        internal override async Task<StepResult> ExecuteAsync(ChainedBlock nextChainedBlock, CancellationToken token, bool disposeMode)
         {
             if (disposeMode)
-                return BlockStoreLoopStepResult.Break();
-
-            var result = BlockStoreLoopStepResult.Next();
+                return StepResult.Stop;
 
             var context = new BlockStoreInnerStepContext(token, this.BlockStoreLoop).Initialize(nextChainedBlock);
 
             this.BlockStoreLoop.BlockPuller.AskBlock(nextChainedBlock);
 
             while (!token.IsCancellationRequested)
             {
-                foreach (var item in context.Routine.ToList())
+                foreach (var innerStep in context.InnerSteps.ToList())
                 {
-                    var executionResult = await item.ExecuteAsync(context);
-                    if (executionResult.ShouldBreak)
-                        return result;
+                    var innerStepResult = await innerStep.ExecuteAsync(context);
```

Comment:
this and above use of `var` is in violation of rule 10 of existing code style - https://github.com/stratisproject/StratisBitcoinFullNode/blob/master/Documentation/coding-style.md